modules/microvm/common/storagevm.nix

```diff
@@ -150,12 +160,11 @@ in
       };
       virtualisation.fileSystems.${cfg.mountPath}.device = "/dev/vda";
 
-      microvm.shares = [
+      microvm.volumes = [
         {
-          tag = "hostshare";
-          proto = "virtiofs";
-          securityModel = "passthrough";
-          source = "/persist/storagevm/${cfg.name}";
+          image = "/persist/storagevm/img/${cfg.name}.img";
+          size = cfg.maximumSize;
+          autoCreate = true;
```

Comment:
do we really want to do this? How to check the integrity of the image? How does update work?  https://github.com/microvm-nix/microvm.nix/blob/1d05a3c26dbb9d4b1cd644e10713a70d8740cb6a/lib/volumes.nix#L60

seems you would need to do a clean of the storagevm/img dir on every nixos rebuild